collector/indices_mappings_response.go

```diff
@@ -0,0 +1,34 @@
+package collector
+
+// IndicesMappingsResponse is a representation of elasticsearch mappings for each index
+type IndicesMappingsResponse map[string]IndexMapping
+
+// IndexMapping defines the struct of the tree for the mappings of each index
+type IndexMapping struct {
+	Mappings Mappings `json:"mappings"`
+}
+
+// Mappings defines all index mappings
+type Mappings struct {
```

Comment:
I think the names `Mappings`, `Properties`, `Fields`, `Property`, and `Field` are really generic and could cause naming collisions in the future. It probably makes sense to use more specific names like `IndexMappings`, `IndexMappingField`, etc.